### PR TITLE
Feat/login

### PR DESCRIPTION
**Pull request by rubyade**
added some code on Index.js to allow a user access to his workspace if he has only one workspace.

### CODE REVIEW

**Comment by ickynavigator** [comment]
resolve conflicts

**Review by ickynavigator — CHANGES_REQUESTED** [review]
two const declarations in card list.js

**Comment by rubyade** [comment]
login
login2
login3
login4

File: topbar/src/TopNavBar.js [hunk 1]
```diff
@@ -172,6 +172,49 @@ const TopNavBar = ({ userProfile: { last_name, first_name } }) => {
   // })
 
   return (
+    <TopNavBarBase>
```
Comment (pidoxy) [inline topbar/src/TopNavBar.js:175]: This conflict was not properly resolved

**Pull request closed without merging**